### ISSUE TITLE
Missing org.eclipse.ui.internal.Shell 

### ISSUE THREAD

**Issue by kciejek** [iloveeclipse/anyedittools#117]
Hi

I want to compile and run application but Eclipse can't find in **src\de\loskutov\anyedit\AnyEditToolsPlugin.java**
class **org.eclipse.ui.internal.Shell**
that should be in org.eclipse.ui.workbench.

My version is **org.eclipse.ui.workbench_3.107.1.v20160120-2131.jar.** 
Eclipse is Version: Mars.2 Release (4.5.2), Build id: 20160218-0600

I will grateful for your help.

kciejek

**Comment by iloveeclipse** [comment]
Sorry, I do not provide courses for Eclipse development due the lack of time.

**Comment by kciejek** [comment]
I've resolved problem and I wanted to share with solution.

It was about org.eclipse.swt.widgets.Shell not org.eclipse.ui.internal.Shell.

I wanted to run AnyEditToolsPlugin on 64 bit Java version what is wrong due to _java.lang.NoClassDefFoundError: org/eclipse/swt/SWTError_ I've received at run.
I've changed to 32 bit Java but then I've got heap size error cause _-Xmx_ was set above 1 GB(limit for 32 bit Java). 
I've set _-Xmx1g_ but don't have set _Target Platform_ in _Plug-in development_ in Eclipse Preferences. 
After correct this AnyEditToolsPlugin run.
